pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -248,6 +249,7 @@ func validateMutatingWebhookConfiguration(e *admissionregistration.MutatingWebho
 		if opts.requireUniqueWebhookNames && len(hook.Name) > 0 {
 			if hookNames.Has(hook.Name) {
 				allErrors = append(allErrors, field.Duplicate(field.NewPath("webhooks").Index(i).Child("name"), hook.Name))
+				continue
 			}
 			hookNames.Insert(hook.Name)
```

Comment:
putting the `Insert(...)` in an `else` block seems better than a `continue`. The `continue` is correct today but could introduce a bug in the future if more validation was added after the `if opts.requireUniqueWebhookNames` block

same comment below

Reply:
done, please check again, thanks 😄